library/src/main/java/io/ona/kujaku/exceptions/LocationComponentInitializationException.java

```diff
@@ -0,0 +1,10 @@
+package io.ona.kujaku.exceptions;
+
+/**
+ * @author Vincent Karuri
+ */
+public class LocationComponentInitializationException extends RuntimeException {
```

Comment:
IMO, this tends to be a bit dangerous unless really crucial that the whole application crashes while throwing this exception. Would you agree with this article https://docs.oracle.com/javase/tutorial/essential/exceptions/runtime.html?

Reply:
I had thought about this. My reasoning is:

```
One of the most important features of the GeoWidget is providing location services to the host app.

If the LocationComponent is wrongly initialized, it will affect the experience on the whole app and make it unusable for this purpose.

Basically as bad as performing operations on a null value, don't you think?
```

Reply:
This has since been removed.